pkg/kubelet/kubelet_pods_test.go

```diff
@@ -184,11 +184,23 @@ func TestMakeMounts(t *testing.T) {
 
 func TestNodeHostsFileContent(t *testing.T) {
```

Comment:
in `ensureHostsFile`, `nodeHostsFileContent` is only called when `hostNetwork=true`. so this is already testing the case.